website/src/graphql/dailp/queries.graphql

```diff
@@ -497,3 +498,16 @@ mutation DeleteContributorAttribution(
 mutation UpdateDocumentMetadata($document: DocumentMetadataUpdate!) {
   updateDocumentMetadata(document: $document)
 }
+
+mutation PostComment($input: PostCommentInput!) {
+  postComment(input: $input) {
+    ... on AnnotatedForm {
+      __typename
+      id
+    }
+    ... on DocumentParagraph {
+      __typename
+      id
+    }
+  }
+}
```

Comment:
_When you do the comments read/delete stuff_ you will want to make sure this query resolves the `comments` field with a sensible fragment so that our frontend will have up-to-date data when we get the mutation response, and we don't have to refetch a comment query or something.